.changeset/orange-students-lie.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/hydrogen': patch
+'@shopify/cli-hydrogen': patch
```

Comment:
```suggestion
```
lolol you can remove now since we've updated our init command to use the GH repo 🙈